overview.md

```diff
@@ -0,0 +1,158 @@
+# SourceCred
+
+SourceCred is anchored by a simple goal: we want open-source contributors to
+earn a living from their work. At first, we focused on the obvious problem:
+where will the money come from? We were inspired by the ability of
+cryptocurrencies to fund open-source projects, and started [SourceGrain], with
+the goal of letting every open-source project issue its own token.
+
+However, as we evolved this idea, we ran into a big problem: we don't know who
```

Comment:
> ~as we evolved this idea~ 

_evolve_ is an intransitive verb. alternatives:
- as this idea evolved"
- as we iterated on this idea
- as we explored this idea"